### PR TITLE
Added fare collection data standard

### PR DESCRIPTION
**Pull request by upperwal**

### CODE REVIEW

File: transit/fare_collection/FareCollection.proto [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    // Indicates the specific physical/digital  form of the ticket.
+    enum TicketType {
+        TICKET_TYPE_UNKNOWN = 0;
+        TICKET_TYPE_PAPER = 1;
+        TICKET_TYPE_TOKEN = 2;
+        TICKET_TYPE_TRAVEL_CARD = 3;
+        TICKET_TYPE_MOBILE = 4;
+    }
+
+    // In case the TIcket Type is a Travel Card, indicates the
```
Comment (MDFCP) [inline transit/fare_collection/FareCollection.proto]: Change TicketType and TravelCard

File: transit/fare_collection/FareCollection.proto [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    // Identifies the final fare charged to the user, 
+    // specifying the various sub-prices accordingly 
+    // with the fare structure utilised. 
+    message Fare {
+        float baseFare = 1;
+        float surchargeFare = 2;
+        float waitingFare = 3;
+        float commissionfare = 4;
```
Comment (MDFCP) [inline transit/fare_collection/FareCollection.proto]: commissionFare

File: transit/fare_collection/FareCollection.proto [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    string id = 1;
+    string cardId = 2;
+    uint64 timestamp = 3;
+    TicketType ticketType = 4;
+    TravelCardType travelCardType = 5;
+    ModeOfPayment modeOfPayment = 6;
+    TicketIssueLocation ticketIssueLocation = 7;
+    TicketVendorType ticketVendorType = 8;
+    Fare fare = 9;
```
Comment (upperwal) [inline transit/fare_collection/FareCollection.proto:181]: Add a new field to Fare.

**Review by MDFCP — APPROVED** [review]

File: common/Header.proto [hunk 4]
```diff
@@ -0,0 +1,50 @@
+// MIT License
```
Comment (aparna237) [inline common/Header.proto:1]: Test comment

**Review by MDFCP — APPROVED** [review]

**Pull request merged**